javascript/react-js/hooks.md

```diff
@@ -91,7 +91,7 @@ export default App;
 ~~~
 Try it out to get an idea of what is happening. 
  
-Once you've done that, let's go through it in all detail.
+Once you've done that, let's go through it all, in detail.
```

Comment:
I don't think the comma is correct here, but "in" and "all" were definitely swapped.

Reply:
You're right. I removed the comma.